src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User | null,
```

Comment:
For previous comment ^
```suggestion
  user: User,
```